### PR TITLE
Merge get/list env command

### PR DESCRIPTION
**Pull request by shivam-Purohit**
Merges both `get chaos-environment` and `list chaos-envrionments` commands into a single command as` get chaos-environments`.

### CODE REVIEW

**Comment by shivam-Purohit** [comment]
@SarthakJain26 its hard to just add a flag and show the whole details. Like not hard but unnecessary cause it will make the command slower if we query all the fields for list environments also. We should consider adding it to descibe or something. WDYT?

**Comment by shivam-Purohit** [comment]
i havent touched other part of the code which can be optimised with this new get environment endpoint, we would do that in some other pr. I will create an issue regarding that.

**Comment by shivam-Purohit** [comment]
```
/litmusctl get chaos-environments --project-id="8adf62d5-64f8-4c66-ab53-63729db9dd9a" --environment-id="shivamenv"

CHAOS ENVIRONMENT DETAILS

CHAOS ENVIRONMENT ID	 shivamenv

CHAOS ENVIRONMENT NAME	 shivamenv

CHAOS ENVIRONMENT Type	 NON_PROD

CREATED AT		 55908-04-03 16:42:51 +0530 IST

CREATED BY		 admin

UPDATED AT		 55908-04-03 16:42:51 +0530 IST

UPDATED BY		 admin

CHAOS INFRA IDs	 d99c7d14-56ef-4836-8537-423f28ceac4e

TAGS	
```
```
./litmusctl get chaos-environments --project-id="8adf62d5-64f8-4c66-ab53-63729db9dd9a"

CHAOS ENVIRONMENT ID	CHAOS ENVIRONMENT NAME		CREATED AT			CREATED BY

shivamnewenv		shivamnewenv			55962-10-01 15:05:45 +0530 IST	admin

newenvironmenttest	newenvironmenttest		55912-12-01 10:55:23 +0530 IST	admin

shivamenv		shivamenv			55908-04-03 16:42:51 +0530 IST	admin

Press Enter to show more environments (or type 'q' to quit): q
```

File: pkg/cmd/get/environments.go [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+			utils.White_B.Print("\nEnter the Project ID: ")
+			fmt.Scanln(&projectID)
+
+			for projectID == "" {
+				utils.Red.Println("⛔ Project ID can't be empty!!")
+				os.Exit(1)
+			}
+		}
+
+		environmentList, err := environment.ListChaosEnvironments(projectID, credentials)
```
Comment (Nageshbansal) [inline pkg/cmd/get/environments.go:54]: Shouldn't we have a check here to see if `environmentID` is given or not before sending the query

File: pkg/cmd/get/get.go [hunk 2]
```diff
@@ -35,6 +35,9 @@ var GetCmd = &cobra.Command{
 		#get list of Chaos Experiment runs
 		litmusctl get chaos-experiment-runs --project-id=""
 
+		#get list of Chaos Environments
+		litmusctl get chaos-experiments --project-id=""
```
Comment (Nageshbansal) [inline pkg/cmd/get/get.go:39]: add example for environment-id also

**Review by Nageshbansal — CHANGES_REQUESTED** [review]

**Comment by shivam-Purohit** [comment]
@Nageshbansal does the master branch have updated-dependencies? Not rebase in case, dont want to add extra commit

**Review by SarthakJain26 — APPROVED** [review]

**Review by Nageshbansal — APPROVED** [review]
LGTM, geat work 🚀

**Pull request merged**